src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -3116,6 +3117,9 @@ void WriteErrorHelper(Exception exception)
                             _removedFiles++;
                             _removedBytes += fileBytesSize;
                             double speed = _removedBytes / 1024 / 1024 / _removeStopwatch.Elapsed.TotalSeconds;
+
+                            if (_removeStopwatch.Elapsed.TotalSeconds > progressBarDurationThreshold)
```

Comment:
Please follow our code style guideline - const should start with uppercase letter:
```suggestion
                            if (_removeStopwatch.Elapsed.TotalSeconds > ProgressBarDurationThreshold)
```